src/Common.props

```diff
@@ -0,0 +1,33 @@
+﻿<Project>
+  <PropertyGroup>
+    <Authors>Progress Onderwijs</Authors>
+    <Owners>Progress Onderwijs</Owners>
+
+    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+    <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
+    <RepositoryUrl>https://github.com/progressonderwijs/ProgressOnderwijsUtils</RepositoryUrl>
+    <PackageProjectUrl>https://github.com/progressonderwijs/ProgressOnderwijsUtils</PackageProjectUrl>
+    <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
+
+    <DefineConstants>TRACE;JETBRAINS_ANNOTATIONS</DefineConstants>
+    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
+    <OutputType>library</OutputType>
+    <LangVersion>8.0</LangVersion>
+    <Nullable>enable</Nullable>
+    <NoWarn>CS1591</NoWarn>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <!-- Source link: -->
+    <DebugType>portable</DebugType>
+    <DebugSymbols>True</DebugSymbols>

```

Comment:
Ik denk dat dit de default is.

Reply:
Not specified here: https://docs.microsoft.com/en-us/visualstudio/msbuild/common-msbuild-project-properties?view=vs-2019 - but might be.  Might also differ between release/debug builds.

Reply:
https://github.com/Microsoft/msbuild/issues/2169
https://github.com/microsoft/msbuild/issues/2170

Reply:
Ik stem voor nog even zo laten gezien hoe slecht dit gedocumenteerd is; wie weet valt het volgende versie weer om.